app/MMB/src/scenes/timeline/TimelineRow.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, Color } from '@kiwicom/mobile-shared';
+
+type Props = {|
+  +leftColumn?: React.Element<typeof View>,
+  +rightColumn?: React.Element<typeof View>,
+  +rowStyle?: Object,
```

Comment:
I think there is a type for it in the shared package.

Reply:
I had to modify the eslint config to support `import` and `import type` for the same file.

Reply:
What? Why? We do it everywhere without eslint changes:

```js
import { Component, type ComponentType } from '../..'
```

Reply:
Oh, right. I'll try again like this then. I think I had tried that though.

Reply:
Well I probably did not try that hard enough... It works. I will revert `eslintrc.js`.